src/System.Windows.Forms/src/System/Windows/Forms/Controls/PictureBox/PictureBox.cs

```diff
@@ -464,23 +464,29 @@ public void Load()
         try
         {
             DisposeImageStream();
-            if (UseWebRequest())
+            Uri uri = CalculateUri(_imageLocation);
+            if (uri.IsFile)
             {
-                LoadImageViaWebClient();
+                _localImageStreamReader = new StreamReader(uri.LocalPath);
+                Image img = Image.FromStream(_localImageStreamReader.BaseStream);
+                InstallNewImage(img, ImageInstallationType.FromUrl);
             }
-            else
+            else if (UseWebRequest())
             {
-                Uri uri = CalculateUri(_imageLocation);
-                if (uri.IsFile)
+                Task.Run(async () =>
                 {
-                    _localImageStreamReader = new StreamReader(uri.LocalPath);
-                    Image img = Image.FromStream(_localImageStreamReader.BaseStream);
+                    HttpResponseMessage response = await _httpClient.GetAsync(uri).ConfigureAwait(false);
+                    response.EnsureSuccessStatusCode();
+
+                    using Stream stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    _uriImageStream = stream;
```

Comment:
This seems equivalent to `HttpClient.GetStreamAsync`
```suggestion
                    _uriImageStream = await _httpClient.GetStreamAsync(uri);
```